welcomescreen.py

```diff
@@ -60,28 +57,32 @@ def run(self, g):
                 # totaltime+=timer.tick()
                 if event.type == pygame.QUIT:
                     crashed = True
-                if event.type == pygame.KEYDOWN and (event.key == pygame.K_LEFT or event.key == pygame.K_RIGHT):
-                    return
+                if event.type == pygame.KEYDOWN:
+                    if event.key == pygame.K_LEFT or \
+                            event.key == pygame.K_RIGHT:
```

Comment:
This can be better written as;
```python3
if event.key == pygame.K_LEFT or pygame.K_RIGHT:
```

Reply:
Oh yeah true, fixed

Reply:
Wait, those two are not the same, no? `pygame.K_RIGHT` will always be true because pygame.K_RIGHT  is a non-zero integer.

Reply:
The comparison is for equality hence `==` and not for identity which is done with `is` as it compares if both objects are the same while the equality compares two values.

Reply:
Your point about `==` checking for equality and `is` checking for identity is accurate, but the issue in the original statement if `event.key == pygame.K_LEFT or pygame.K_RIGHT:` is with the use of the `or` operator. The way it's written, Python evaluates it as `(event.key == pygame.K_LEFT)` or `(pygame.K_RIGHT)`. Since `pygame.K_RIGHT` is a non-zero integer constant, it evaluates as `True` in a boolean context, making the entire or condition always `True`. Therefore, this condition doesn't accurately check `if event.key` is either `pygame.K_LEFT or pygame.K_RIGHT`. 

When this code is used, the game starts only when the left or the right arrow keys are pressed
```
if event.key == pygame.K_LEFT or event.key == pygame.K_RIGHT:
```

When this code is used, the game starts when any key is pressed
```
if event.key == pygame.K_LEFT or pygame.K_RIGHT:
```

Reply:
I agree.  But easier to demonstrate;

```
>>> import pygame
>>> x = pygame.K_DOWN
>>> x == pygame.K_LEFT or x == pygame.K_RIGHT
False
>>> x == pygame.K_LEFT or pygame.K_RIGHT
1073741903
>>> if 1073741903:
...     print('true')
... 
true
>>> 
```